CHANGELOG.md

```diff
@@ -9,6 +9,9 @@
 - Updated deserialization of `bitmap` for custom CPU templates to allow usage
   of '_' as a separator.
 - Changed the strip feature of `cpu-template-helper` tool to operate bitwise.
+- Better logs during validation of cpu id in snapshot restoration path. Also
```

Comment:
nits:
- `s/cpu/CPU/`
- `s/id/ID/`
- `s/can not/cannot/`

Reply:
done